### PR TITLE
Reformat files, upgrade node version, lock npm version, upgrade dependencies

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +16,8 @@
     "start": "cross-env REACT_APP_VERSION=$npm_package_version react-scripts start",
     "test": "react-scripts test",
     "lint": "eslint . --ext .ts,.tsx --max-warnings=0",
-    "lint:fix": "eslint . --fix --ext .ts,.tsx"
+    "lint:fix": "eslint . --fix --ext .ts,.tsx",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\""
```
Comment (polarker) [inline package.json:20]: 👍

**Comment by nop33** [comment]
It seems that it's again hanging after removing node version 12:

image

**Comment by polarker** [comment]
> It seems that it's again hanging after removing node version 12:
> 
> image

Updated the setting. Github is not smart enough for action changes

**Comment by nop33** [comment]
Got the approval of @mvaivre on Slack. Merging...

**Pull request merged**